python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -283,6 +345,12 @@ def get_snfluxes(self):
                 vals[:, 16], vals[:, 17], vals[:, 18])
 
 
+    def get_EBV(self):
```

Comment:
Hmm ... I am not sure why what I have works (or does it)?

The question about using the EBVmixin is what coordinates would be used for a SN, which is tied to a galaxy which are at different positions? In practice this is not much of an issue for what we have now But we already have talked about making `hostless` SN for Twinkles3.